extras/parse_travis_log.py

```diff
@@ -899,7 +900,6 @@ def build_return_code(status_vera_init,
        (status_make_install) and \
        (status_tests) and \
        (number_of_msgs_in_summary(summary_vera) == 0) and \
-       (number_of_msgs_in_summary(summary_cppcheck) == 0) and \
```

Comment:
@heplesser, @jougs finally, I ended up with adding additional logic to be able to ignore certain static code analysis messages or skip the analysis.  (5be3fda) This will then be reported with 'Ignored' or 'Skipped', respectively. I found this is the best solution and provides a central point of configuration in 'build.sh'. This information is passed through the log and parsed by the Python parser script.

The current setting is:
- all checks are turned on
- CPPCHECK messages are ignored

I also refactored some of the function names.